frontend/src/booking/booking-modal.vue

```diff
@@ -315,11 +347,27 @@
         }
         this.notes = this.exam.notes.valueOf()
       },
+      clickNext() {
+        this.formStep = 2
+        console.log(this.invigilatorId)
+        console.log(this.clickedRow)
+        if (!this.invigilatorId && this.clickedRow) {
+          console.log('we did it')
+          this.clickedRow._rowVariant = ''
+        }
+      },
       rowClicked(item) {
-        this.selectedInvigilator = item.invigilator_id
+        console.log(item)
```

Comment:
Console log

Reply:
i'd like to someday submit a PR free extraneous console.logs.  someday...